test/form-layout.html

```diff
@@ -545,6 +545,24 @@
         expect(estimateEffectiveColspan(layout.children[0])).to.be.closeTo(2, .1);
       });
 
+      it('should update style after layout-item is not hidden', async() => {
+        sinon.spy(layout, '_invokeUpdateStyles');
+        const firstDisplay = layout.firstElementChild.style.display;
+        layout.firstElementChild.setAttribute('hidden', true);
+        container.shadowRoot.querySelector('vaadin-form-layout').firstElementChild.style.display = 'none';
+        layout.firstElementChild.removeAttribute('style');
+        let isStyleAfterRemovedNullOrEmpty = false;
+        if (layout.firstElementChild.getAttribute('style') === ''
+            || layout.firstElementChild.getAttribute('style') === null) {
+          isStyleAfterRemovedNullOrEmpty = true;
+        }
```

Comment:
`isStyleAfterRemovedNullOrEmpty` can never end up being anything else than `true` because you just removed all the inline styles one row above

Reply:
changed test